test/foundry/forking/REP-002/UnattachBridgeForkTest.t.sol

```diff
@@ -76,9 +139,21 @@ contract UnattachBridgeForkTest is RoninTest {
     ICoinbaseExecution(address(RONIN_VALIDATOR_SET_CONTRACT)).wrapUpEpoch();
 
     assertEq(_getProxyImplementation(RONIN_VALIDATOR_SET_CONTRACT), _newImpl);
+    assertEq(
+      _getProxyImplementation(_roninTrustedOrgProxy),
+      NotifiedMigrator(payable(_roninTrustedOrgSwitcher)).NEW_IMPL()
+    );
+    assertEq(_getProxyImplementation(_stakingProxy), NotifiedMigrator(payable(_stakingSwitcher)).NEW_IMPL());
+    assertEq(
+      _getProxyImplementation(_slashIndicatorProxy),
+      NotifiedMigrator(payable(_slashIndicatorSwitcher)).NEW_IMPL()
+    );
   }
 
   function _upgradeToVersionSwitcher() internal fromWho(_getProxyAdmin(RONIN_VALIDATOR_SET_CONTRACT)) {
     RONIN_VALIDATOR_SET_CONTRACT.upgradeTo(_versionSwitcher);
+    _roninTrustedOrgProxy.upgradeTo(_roninTrustedOrgSwitcher);
+    _stakingProxy.upgradeTo(_stakingSwitcher);
+    _slashIndicatorProxy.upgradeTo(_slashIndicatorSwitcher);
```

Comment:
Write a few lines that interact with the upgraded contracts to assure it works. Test on a few sample methods, especially calling the methods that are changes in the upgrade.

Reply:
updated.